checks.tf

```diff
@@ -0,0 +1,10 @@
+// ensure that organization_id is not empty, even for project-level integrations
```

Comment:
Please update the examples for project-level integration if org ID is required for them also.

Reply:
actually I worded it badly. It should be "ensure that local.organization_id is not empty...". Even though in project-level integration examples we leave `var.organization_id` empty, it's still fine because we should be able to infer `local.organization_id` from the `project_id` provided via TF providers. I'll update the comment.